roles/ceph-common/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
So this is going to be interesting. If we're wholesale copying files, I think we'd have to keep the apache license for them and mark which files fall under this. However if we use the upstream files as inspiration but write them ourselves, we could default to the MIT license the rest of the tree is in. However I'm not a lawyer. Dual-licensing is tough, and I'd prefer that we don't do that.